ProjectLighthouse/ProjectLighthouse.csproj

```diff
@@ -15,17 +15,17 @@
           <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
         <PackageReference Include="Pfim" Version="0.11.2" />
-        <PackageReference Include="SixLabors.ImageSharp" Version="3.0.2" />
-        <PackageReference Include="Discord.Net.Webhook" Version="3.12.0" />
-        <PackageReference Include="JetBrains.Annotations" Version="2023.2.0" />
-        <PackageReference Include="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Version="7.0.13" />
-        <PackageReference Include="Microsoft.EntityFrameworkCore" Version="7.0.13" />
-        <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="7.0.13">
+        <PackageReference Include="SixLabors.ImageSharp" Version="3.1.1" />
+        <PackageReference Include="Discord.Net.Webhook" Version="3.13.0" />
+        <PackageReference Include="JetBrains.Annotations" Version="2023.3.0" />
+        <PackageReference Include="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Version="8.0.0" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore" Version="8.0.0" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="8.0.0">
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
-        <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" Version="7.0.0" />
-        <PackageReference Include="Redis.OM" Version="0.5.4" />
+        <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" Version="8.0.0-beta.2"/>
```

Comment:
We should maybe wait for the final 8.0 release of the EF driver for MySql

Reply:
It seems this should be coming soon.

> First of all: Happy new year everybody!
>
> I see there is some discussion going on here. 😇
>
> Unfortunately, the original plan did not survive contact with my actual real life business obligations. The new plan is to push all open features to the backlog, check and fix any critical bugs reported for 8.0.0-beta.2 and then directly go GA with 8.0.0.
>
> Thanks for the patience.